### PR TITLE
Generating toml file

### PR DESCRIPTION
**Pull request by albertmena**
Issue related: https://github.com/I2PC/scipion-em-xmipp/issues/826
Was imperative to move the version details to an external file (__init__ has to much imports and the pip tool fails), Maybe there is  other option but I can not spend more time on it. This proposal works.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "scipy<=1.10.0",
+    "joblib",
+    "peppercorn",
+    "plotter",
+    "PyWavelets",
+]
+
+authors = [
+    {name = "Oier Lauzirika Zarrabeitia", email = "oierlauzi@bizkaia.eu"},
+    {name = "Alberto Garcia Mena", email = "pconesa@cnb.csic.es"},
```
Comment (oierlauzi) [inline pyproject.toml:23]: Change e-mail

**Review by oierlauzi — CHANGES_REQUESTED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scipion-em-xmipp"
+dynamic = ["version"]
+dependencies = [
+    "scipion-em",
+    "scipion-pyworkflow>=3.0.31",
+    "numpy>=1.21.0",
```
Comment (pconesa) [inline pyproject.toml:11]: I think we should not define a dependency already in the core

**Comment by albertmena** [comment]
Reverting changes because of the logo issue (in release it does not appear because an issue in the scipion toml integration)

**Comment by albertmena** [comment]
scikit-image is going to appear as dependency: https://github.com/I2PC/scipion-em-xmipp/pull/838/files

**Review by oierlauzi — APPROVED** [review]
AFAIK everything is OK for me, but I am not very familiar with pyproject.toml files.

**Comment by albertmena** [comment]
> AFAIK everything is OK, but I am not very familiar with pyproject.toml files.

in any case it is not implemented for your current use (rename the project.toml)

**Pull request merged**